LoopKit/TherapySettings.swift

```diff
@@ -121,7 +121,7 @@ extension TherapySettings: Codable {
         let insulinSensitivitySchedule = try container.decodeIfPresent(InsulinSensitivitySchedule.self, forKey: .insulinSensitivitySchedule)
         let carbRatioSchedule = try container.decodeIfPresent(CarbRatioSchedule.self, forKey: .carbRatioSchedule)
         let basalRateSchedule = try container.decodeIfPresent(BasalRateSchedule.self, forKey: .basalRateSchedule)
-        let insulinModelSettings = try container.decodeIfPresent(InsulinModelSettings.self, forKey: .insulinModelSettings)
+        let defaultRapidActingModel = try container.decodeIfPresent(ExponentialInsulinModelPreset.self, forKey: .defaultRapidActingModel)
```

Comment:
I don't think this is backward-compatible.  Existing Tidepool Loop users will need to either realize this, or delete and install fresh.  Please make sure to make an announcement when a build with this change is published.